src/display/annotation_layer.js

```diff
@@ -190,7 +190,7 @@ class AnnotationElement {
    * @private
    * @param {boolean} ignoreBorder
    * @memberof AnnotationElement
-   * @returns {HTMLSectionElement}
+   * @returns {HTMLElement} A section element
```

Comment:
Here, and similarly below, please put a period at the end since it's now a complete sentence.
```javascript
 * @returns {HTMLElement} A section element.
```